traffic/TrafficCounter.py

```diff
@@ -0,0 +1,132 @@
+import paho.mqtt.publish as publish
+import grovepi
+import json
+from time import sleep
+from configparser import ConfigParser 
+
+
+
+# NOTE:
+# The sensitivity can be adjusted by the onboard potentiometer
+# Calibration
+#       Hold the sensor facing an empty dark area
+#       Adjust the potentiometer until the onboard LED switches off
+#       When the LED switches off the sensor is calibrated
+#       If not, adjust the potentiometer again
+#IMPORTANT: When assigning the port to the sensor assign the next port
+#           from where the sensor has been connected
+#           e.g.: if sensor connectet to d4 assign the sensor to port d5
+#           otherweise the sensor won't work. also leave port d5 and d3 free
+#           to ensure the sensor works correctly
+
+# Connect the Grove Infrared Distance Interrupt Sensor (ir) to digital port D5
+# (see above note)
+
+##read values from config ini   
+file = 'config.ini'
+config = ConfigParser()
+config.read(file)
+
+
+##topic constructor
+def topic_constructor(protocol, service_api_key, sensor_id):
+   return ("/" + protocol + "/" + service_api_key + "/" + sensor_id)
+
+def attr_topic(topic):
+    return (topic + "/attrs")
+
+def cmd_topic(topic):
+    return (topic + "/cmd")
+
+def ack_topic(cmd_topic):
+    return (cmd_topic + "exe")
+
+
+#config connection
+broker_address=str(config['connection']['broker_address'])
+port = int(config['connection']['port'])
+username = str(config['connection']['username'])
+password = str(config['connection']['password'])
+##authentication object needed to publish on topic
+auth = {'username':username, 'password':password}
+##topic base config
+protocol = "json" ## str(config['connection']['protocol'])
+service_api_key = "test_key"#str(config['Street 2']['service_api_key'])
+
+##ir Sensor data and functions
+##Connect the ir sensor at port d6 NOT d7 (see initial note)
+ir_pin = 7 #str(config['Street 1']['ir_pin'])
+grovepi.pinMode(ir_pin,"INPUT")
+ir_id = "ir" #str(config['Street 1']['ir_id'])
+ir_topic = topic_constructor(protocol, service_api_key, ir_id)
+# ir_car_counter = 0
+
+
+
+
+## Obstacle sensor data and function
+## connect the obstacle sensor to port d8
+os_pin = 8
+grovepi.pinMode(os_pin,"INPUT")
+os_id = "os" #str(config['Street 2']['os_id'])
```

Comment:
load form configuration (don't use spaces) so "street2"

Reply:
done config.ini was defind in issue #19